CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -32,6 +32,7 @@
 - Fixes an issue where redundant CSIStorageCapacity objects might have been created because informers were not handled correctly. ([#590](https://github.com/kubernetes-csi/external-provisioner/pull/590), [@bai3shuo4](https://github.com/bai3shuo4))
 - Fixes race where producing StorageCapacity objects may have failed with `the object has been modified` errors ([#565](https://github.com/kubernetes-csi/external-provisioner/pull/565), [@pohly](https://github.com/pohly))
 - If the in-tree providers are depending on StorageClass for DeleteVolume, volumes could not get deleted after migrating from them. So translate in-tree StorageClass to CSI to get SecretReference in DeleteVolume call for CSI Migration scenario. ([#567](https://github.com/kubernetes-csi/external-provisioner/pull/567), [@huchengze](https://github.com/huchengze))
+- When using immediate binding, allowed topologies and distributed provisioning, PVCs may have been assigned to nodes which did not fit the allowed topologies, leading to permanent `error generating accessibility requirements` failures. ([#612](https://github.com/kubernetes-csi/external-provisioner/pull/612), [@pohly](https://github.com/pohly))
```

Comment:
I just reworded this. Can you update it again?

Reply:
Done